### DISCUSSION TITLE
What is default front-end for this

### DISCUSSION THREAD

**Discussion by abdulbadii** [fairy-stockfish/Fairy-Stockfish#806] [Q&A]
What is the default/most suitable front-end for this (where's the link) i.e. particularly of the non 8x8 board such as capablanca is there special protocol on such ?

**Comment by gbtami** [comment]
You can find offline/online GUI links on https://fairy-stockfish.github.io/

**Comment by PraseodymiumSpike** [comment]
> What is the default/most suitable front-end for this (where's the link) i.e. particularly of the non 8x8 board such as capablanca is there special protocol on such ?

there is fairyground
